lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -1073,8 +1079,21 @@ RuleToProcess SessionState::make_rule_to_process(const PolicyRule& rule) {
   RuleToProcess to_process;
   to_process.version = get_current_rule_version(rule.id());
   to_process.rule    = rule;
-  // TODO(@themarwhal): look up teids from PolicyID -> BearerID map
-  to_process.teids = config_.common_context.teids();
+  to_process.teids   = config_.common_context.teids();
+
+  // At this point, we know the rule exists, so just check if it exists in the
+  // static rule store or not
+  bool is_static    = static_rules_.get_rule(rule.id(), nullptr);
+  PolicyType p_type = is_static ? STATIC : DYNAMIC;
+
+  // If there is a dedicated bearer TEID already in map, use it
+  PolicyID policy_id = PolicyID(p_type, rule.id());
+  bool bearer_exists =
```

Comment:
shouldnt this be `dedicated_bearer_exists` or is this also finding default bearer?

Reply:
yeah you are right it's better to be specific.